### ISSUE TITLE
Missing CSS class feedbackBorderERROR for FormComponentFeedbackBorder component

### ISSUE THREAD

**Issue by tverdokhlebd** [sebfz1/wicket-jquery-ui#344]
CSS class "feedbackBorderERROR" is missing for FormComponentFeedbackBorder component in Wicket 9.x version.
Wicket 8 uses just inline-style "style="color:red;", however wicket 9 uses CSS class "feedbackBorderERROR" that cannot be found.

**Comment by martin-g** [comment]
The idea in the change in Wicket 9.x is to use this CSS class as a "placeholder".
It should be provided by the application depending on its UI requirements.

But maybe libraries like Wicket jQuery UI and Wicket Bootstrap should/could provide it as well, since they provide UI styling already.

**Comment by sebfz1** [comment]
Right, FormComponentFeedbackBorder is not part of Wicket jQuery UI, as such
it doesn't provide the styling class. Maybe you can use the dedicated
FeedbackPanel intead
As for providing the styling infos for this component, both jQuery UI and
Kendo UI provides their own styling classes, with a different name
obviously. Maybe I can try to import these styling into feedbackBorderXxx,
but no guarantee... I will try this asap...

**Comment by tverdokhlebd** [comment]
Thanks for the answers.
I just wanted to say that moving from Wicket 8 to Wicket 9 should not break behavior for this component.
I expected that this style class will contain red color for default behavior.

**Comment by martin-g** [comment]
Some people prefer red border for the input, others red hint text or red asterisk next to the input field, other people prefer a floating red-ish box in to appear at the top of the window, ...
Wicket uses SemVer and the upgrade from 8.x to 9.x is the perfect time to make such breaking (behavior) changes!
Unfortunately https://cwiki.apache.org/confluence/display/WICKET/Migration+to+Wicket+9.0 does not mention it! :-(
